src/app/pages/ProfilePage/components/app-profile/app-details/app-details.component.ts

```diff
@@ -14,4 +14,12 @@ export class AppDetailsComponent {
 
     @Input() appDetails: AppDetails;
 
+    technologies: string[];
+
+    ngOnInit(){
+    	console.log(this);
+    	this.technologies = this.appDetails.technologies.split(',').map(e=>e.trim());
```

Comment:
Trebuie scos `console.log` - exista si [optiune de linting](https://palantir.github.io/tslint/rules/no-console/) pentru asa ceva.

```suggestion
    	this.technologies = this.appDetails && this.appDetails.technologies ? this.appDetails.technologies.split(',').map(e=>e.trim()) : [];
```

Reply:
done (removed trim though, it's not needed)